spring-boot-admin-server-ui/src/main/frontend/views/journal/index.spec.ts

```diff
@@ -0,0 +1,123 @@
+import { mount } from '@vue/test-utils';
+import { HttpResponse, http } from 'msw';
+import { beforeEach, describe, expect, it } from 'vitest';
+
+import JournalView from './index.vue';
+
+import { server } from '@/mocks/server';
+import { render } from '@/test-utils';
+
+class InstanceEvent {
+  constructor({ instance, version, type, timestamp, ...payload }) {
+    this.instance = instance;
+    this.version = version;
+    this.type = type;
+    this.timestamp = new Date(timestamp);
+    this.payload = payload;
+  }
+
+  get key() {
+    return `${this.instance}-${this.version}`;
+  }
+}
+InstanceEvent.REGISTERED = 'REGISTERED';
+InstanceEvent.REGISTRATION_UPDATED = 'REGISTRATION_UPDATED';
+
+describe('Journal View', () => {
+  beforeEach(() => {
+    server.use(
+      http.get('/instances/events', () => {
+        return HttpResponse.json({});
+      }),
+    );
+  });
+
+  it('should update instance names when registration is updated', () => {
+    const events = [
+      new InstanceEvent({
+        instance: 'instance-1',
+        version: 1,
+        type: 'REGISTERED',
+        timestamp: '2023-01-01T10:00:00Z',
+        registration: { name: 'OLD APP NAME' },
+      }),
+      new InstanceEvent({
+        instance: 'instance-1',
+        version: 2,
+        type: 'REGISTRATION_UPDATED',
+        timestamp: '2023-01-01T11:00:00Z',
+        registration: { name: 'NEW APP NAME' },
+      }),
+    ];
+
+    const { container } = render(JournalView, {
+      data() {
+        return {
+          events: events,
+        };
+      },
+      global: {
+        mocks: {
+          $t: (key) => key,
+          $route: { query: {} },
+          $router: { replace: () => {} },
+        },
+      },
+    });
+
+    const instanceNames =
+      container.firstChild.__vueParentComponent.ctx.instanceNames;
```

Comment:
in other tests we do not use properties, but check what is visible on screen, right?

Reply:
maybe @SteKoe should review this before merge